### ISSUE TITLE
 Static route is incorrect when max PortChannels are configured and removed

### ISSUE THREAD

**Issue by JibinBao** [Azure/sonic-buildimage#7261]
#### Description

Firstly create max number of PortChannels depending on system type and remove them. Secondly create one PortChannel, and  add static route with this PortChannel. After that we can find the static route info is not correct, namely the PortChannel is not the specified PortChannel.

#### Steps to reproduce the issue:
1.  Create max number of PortChannels:
      sudo config portchannel add PortChannel1
      sudo config portchannel add PortChannel2
      sudo config portchannel add PortChannel3
      sudo config portchannel add PortChannel4
      ......

      show interfaces portchannel
      sudo config save -y
      reboot
      show interfaces portchannel

2.  Del all PortChannels created by step 1
     sudo config portchannel del PortChannel1
     sudo config portchannel del PortChannel2
     sudo config portchannel del PortChannel3
     .....

3.  Configure route like following example:
     sudo config route add prefix 20.0.0.1/32 nexthop PortChannel0001

3.  Check route info:
      show ip route 20.0.0.1

#### Describe the results you received:
   Known via "static", distance 1, metric 0, best
   Last update 00:00:12 ago
  * directly connected, **PortChannel1**, weight 1

#### Describe the results you expected:
   Known via "static", distance 1, metric 0, best
   Last update 00:00:12 ago
  * directly connected, **PortChannel0001**, weight 1

#### Output of `show version`:

```
SONiC Software Version: .master branch, commit hash: 01b03307


```
Attached below is sysdump log:
[sysdump_test_basic_static_route.tar.gz](https://github.com/Azure/sonic-buildimage/files/6277919/sysdump_test_basic_static_route.tar.gz)

#### Additional information you deem important (e.g. issue happens only occasionally):
(First case: create portchannel. Second case: create static route)
The first test has created a port-channel with the name PortChannel1.

At the end of the test, the cleanup has removed the port-channels the tests have created but did not reload the configuration or the switch.

Apparently, there is a cache inside vtyshs that remembered this port-channel name, and when we added a route to PortChannel0001 it added it under the name of the previously created port-channel (PortChannel1).

Reloading the BGP service recreates the routes with the correct values and the tests pass.

To prevent the tests from failing we can run at the end of the test config reload or change the tests to use the same pattern of the port-channel name. But doing so won't fix the issue that the route was added to a non-existing port-channel, this could be done by forcing to reload the service after removing a port-channel, but we probably don't want to do this like this.

**Comment by maksymbelei95** [comment]
Working on the issue

**Comment by maksymbelei95** [comment]
The issue will be fixed in scope of PR https://github.com/Azure/sonic-utilities/pull/1534 by checking whether the interface for nexthop parameter is present in the system or not before creation of the static route.

**Comment by anshuv-mfst** [comment]
FYI - @d-dashkov, please include this issue in your fix/add this Issue to PR#1534.
